x-pack/plugins/fleet/server/services/agent_policy.ts

```diff
@@ -698,6 +698,23 @@ class AgentPolicyService {
           }),
     };
 
+    // Only add permissions if output.type is "elasticsearch"
+    fullAgentPolicy.output_permissions = Object.keys(fullAgentPolicy.outputs).reduce<
+      NonNullable<FullAgentPolicy['output_permissions']>
+    >((permissions, outputName) => {
+      const output = fullAgentPolicy.outputs[outputName];
+      if (output && output.type === 'elasticsearch') {
+        permissions[outputName] = {};
+        permissions[outputName].fallback = [
```

Comment:
I know it's unlikely that we'll have an integration called `fallback` that would conflict with this :) but what do you think of prefixing this like `_fallback` or even `_default` to be more explicit about the fact that it's a special key?

Reply:
Good point. I'll change it